src/presentation/http/router/noteList.test.ts

```diff
@@ -1,27 +1,40 @@
-import userSessions from '@tests/test-data/user-sessions.json';
+import type User from '@domain/entities/user';
 import { describe, test, expect, beforeAll } from 'vitest';
 
-
-/**
- * Access token that will be used for Auhorization header
- */
 let accessToken = '';
+let user: User;
 
 describe('NoteList API', () => {
-  beforeAll(() => {
+  beforeAll(async () => {
     /**
-     * userId for authorization
+     * Truncate all tables, which are needed
+     * Restart autoincrement sequences for data to start with id 1
+     *
+     * @todo get rid of restarting database data in tests (move to beforeEach)
      */
-    const userId = userSessions[0]['user_id'];
+    await global.db.truncateTables();
 
-    accessToken = global.auth(userId);
-  });
+    /** create test user */
+    user = await global.db.insertUser({
+      email: 'test@codexmail.com',
+      name: 'CodeX',
```

Comment:
use const user indstead

Reply:
const variable will not be able to be modified, which means that it must be defined before `beforeAll`, and the instruction with the declaration is needed directly in it